### PR TITLE
Fixed city string exceptions

### PR DESCRIPTION
**Pull request by MaxShapiro32**

### CODE REVIEW

**Comment by moficodes** [comment]
seems pretty magical fix. what was happening?

File: services/car-rental-v1/src/services/dataHandler.js [hunk 1]
```diff
@@ -4,12 +4,22 @@ import TagNotFoundError from "./../errors/TagNotFoundError";
 
 const CARS_PATH = path.join(__dirname, "./../../data/cars.json");
 
-const capitalize = (text) =>
-  text
+const lowercaseExceptions = ["es", "de", "au"];
```
Comment (moficodes) [inline services/car-rental-v1/src/services/dataHandler.js:8]: add some comment for this logic

**Review by moficodes — CHANGES_REQUESTED** [review]
add some comment to clarify why the exceptions were needed.

File: services/car-rental-v1/src/services/dataHandler.js (hunk 1 above)
Comment (MaxShapiro32) [reply services/car-rental-v1/src/services/dataHandler.js:8]: Done

**Review by moficodes — APPROVED** [review]
lgtm

**Pull request merged**